server/src/main/java/org/candlepin/guice/CandlepinFilterModule.java

```diff
@@ -57,6 +58,7 @@ protected void configureServlets() {
 
         filterRegex(regex).through(CandlepinScopeFilter.class);
         filterRegex(regex).through(CandlepinPersistFilter.class);
+//        filterRegex(noStatusResource).through(CandlepinSuspendModeFilter.class);
```

Comment:
Is this required? Or something that has to be turned on later? If it is something that should be addressed please add a TODO or FIXME. If not, lets remove it.